internal/operator/operator.go

```diff
@@ -1340,11 +1420,15 @@ func (r *minioPolicyBindingReconciler) Reconcile(ctx context.Context, req reconc
 				User:     pb.Spec.User.Builtin,
 			})
 		}
+		if pb.Spec.Migrate {
```

Comment:
Policy bindings _will_ throw an error if the attach operation results in a no-op.  Thus, supporting migrations simply ignores the 'policy change already applied' error when `.spec.migrate` is set.